template/style.css

```diff
@@ -1,131 +1,465 @@
 :root {
-    /* --color1: #05f2af; */
-    --primary: #01ABAA;
-    --secondary: #FEA680;
-    --alert: #FEDC3D;
-    --light: #efefee;
-    --dark: #333;
-    --dark2: #000;
+  /* --color1: #05f2af; */
+  --primary: #01abaa;
+  --secondary: #fea680;
+  --alert: #fedc3d;
+  --light: #efefee;
+  --dark: #333;
+  --dark2: #000;
 }
 
 body {
-    padding: 0;
-    margin: 0;
-    background-color: var(--light);
+  padding: 0;
+  margin: 0;
+  background-color: var(--light);
+}
+:root {
+  /* --color1: #05f2af; */
+  --primary: #01abaa;
+  --secondary: #fea680;
+  --alert: #fedc3d;
+  --light: #efefee;
+  --dark: #333;
+  --dark2: #000;
 }
 
-/* Navbar container */
-.navbar {
-    overflow: hidden;
-    background-color: #333;
-    font-family: Arial;
-  }
-  
-  /* Links inside the navbar */
-  .navbar a {
-    float: left;
-    font-size: 1.2em;
-    font-weight: bolder;
-    background-color: var(--dark);
-    color: var(--primary);
-    text-align: center;
-    padding: 14px 16px;
-    text-decoration: none;
-  }
-  
-  /* The dropdown container */
-  .dropdown {
-    float: left;
-    overflow: hidden;
-  }
-  
-  /* Dropdown button */
-  .dropdown .dropbtn {
-    font-size: 1.2rem;
-    border: none;
-    outline: none;
-    color: var(--dark);
-    padding: 14px 16px;
-    background-color: inherit;
-    font-family: inherit; /* Important for vertical align on mobile phones */
-    margin: 0; /* Important for vertical align on mobile phones */
-  }
-  
-  /* Add a red background color to navbar links on hover */
-  .navbar a:hover, .dropdown:hover .dropbtn {
-    background-color: var(--primary);
-    color: var(--secondary);
-  }
-  
-  /* Dropdown content (hidden by default) */
-  .dropdown-content {
-    display: none;
-    position: absolute;
-    background-color: var(--secondary);
-    min-width: 160px;
-    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
-    z-index: 1;
-  }
-  
-  /* Links inside the dropdown */
-  .dropdown-content a {
-    float: none;
-    color: var(--primary);
-    padding: 12px 16px;
-    text-decoration: none;
-    display: block;
-    text-align: left;
-  }
-  
-  /* Add a grey background color to dropdown links on hover */
-  .dropdown-content a:hover {
-    background-color: var(--primary);
-    color: var(--secondary);
+/* Background Utils */
+.bg-primary {
+  background-color: var(--primary);
+}
+.bg-secondary {
+  background-color: var(--secondary);
+}
+.bg-alert {
+  background-color: var(--alert);
+}
+.bg-light {
+  background-color: var(--light);
+}
+.bg-dark {
+  background-color: var(--dark);
+}
+.bg-dark2 {
```

Comment:
can we come up with a better name? bg-dark2 doesnt really tell me what it is going to be used for